json/default-pt.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://raw.githubusercontent.com/raspberrypi/picotool/jsonschema/json/schemas/partition-table-schema.json",
```

Comment:
Should we ensure that `rp2350` is in the schema-URL somewhere, in case future chips need different schema files?

Reply:
I think the schema would hopefully stay the same for future chips - this isn't really a proper URL anyway, if we wanted to do it properly then we could add GitHub Pages to host the schemas, and then host schemas for each released version of picotool (in case they do change at a later date)

Reply:
Actually, we probably don't need full on GitHub Pages for this - the `raw.githubusercontent` URLs have versioning, so you can just use URLs like this:
```
https://raw.githubusercontent.com/raspberrypi/picotool/master/json/schemas/partition-table-schema.json
https://raw.githubusercontent.com/raspberrypi/picotool/develop/json/schemas/partition-table-schema.json
https://raw.githubusercontent.com/raspberrypi/picotool/2.1.0/json/schemas/partition-table-schema.json
```

I think this is probably the simplest solution - you can stick to a specific version, or track `master` or `develop`

Reply:
> I think the schema would hopefully stay the same for future chips

Ahhh, fair enough. Is that likely to be true for **all** the schemas that you're adding in this PR?

Reply:
> > I think the schema would hopefully stay the same for future chips
> 
> Ahhh, fair enough. Is that likely to be true for **all** the schemas that you're adding in this PR?

The hope would be that they will all be backwards compatible at least - but as I said if they do change then you can pin to a specific version using the URL, without needing any special folder structure.